BASE/src/Microsoft.ApplicationInsights/Channel/InMemoryChannel.cs

```diff
@@ -96,6 +95,20 @@ public TimeSpan SendingInterval
             }
         }
 
+        /// <summary>
+        /// Gets or sets the <see cref="CredentialEnvelope"/> which is used for AAD.
+        /// FOR INTERNAL USE. Customers should use <see cref="TelemetryConfiguration.SetAzureTokenCredential"/> instead.
```

Comment:
We don't really need this comment anymore since ISupportCredentialEnvelope is, itself, internal.